packages/forms/src/directives/validators.ts

```diff
@@ -363,25 +370,20 @@ export class RequiredValidator implements Validator {
   }
 
   set required(value: boolean|string) {
-    this._required = value != null && value !== false && `${value}` !== 'false';
-    if (this._onChange) this._onChange();
+    this._required = toBoolean(value);
   }
 
-  /**
-   * Method that validates whether the control is empty.
-   * Returns the validation result if enabled, otherwise null.
-   * @nodoc
-   */
-  validate(control: AbstractControl): ValidationErrors|null {
-    return this.required ? requiredValidator(control) : null;
-  }
+  /** @internal */
+  override inputName = 'required';
 
-  /**
-   * Registers a callback function to call when the validator inputs change.
-   * @nodoc
-   */
-  registerOnValidatorChange(fn: () => void): void {
-    this._onChange = fn;
+  /** @internal */
+  override normalizeInput = (input: unknown): boolean => toBoolean(input);
+
+  /** @internal */
+  override createValidator = (input: unknown): ValidatorFn => requiredValidator;
+
+  override enabled(): boolean {
+    return this.normalizeInput(this.required) as boolean;
```

Comment:
This should no longer need to call `this.normalizeInput()` since `required` is normalized in the setter.

Reply:
Right, I think this line should just directly reference the `required` field:
```suggestion
    return this.required;
```

Reply:
done but with minor change